sandBox/FirstSprint/TaskFunctions.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+
+
+namespace FirstSprint
+{
+    class TaskFunctions
+    {
+
+        private int TemporaryResult(int firstIndexOfArray, string[] inputArray)
+        {
+            int result = 0;
+
+            for (int i = firstIndexOfArray; i < inputArray.Length; i++)
+            {
+                if (int.Parse(inputArray[i]) > 0)
+                {
+                    result = 1;
+                    break;
+                }
+
+            }
+
+            return result;
+        }
+
+
+        public int CompareVersions(string initialValue, string comparisonValue)
+        {
+            int result = 0;
+            bool flag = false;
```

Comment:
flag variable name should be self explanatory, as at the moment of its declaration its purpose is unknown.